zookeeper-server/src/main/java/org/apache/zookeeper/server/admin/StreamOutputter.java

```diff
@@ -46,7 +46,7 @@ public void output(final CommandResponse response, final OutputStream os) {
         try (final InputStream is = response.getInputStream()){
             IOUtils.copyBytes(is, os, 1024, true);
         } catch (final IOException e) {
-            LOG.error("Exception occurred when streaming out data to {}", clientIP, e);
+            LOG.warn("Exception streaming out data to {}", clientIP, e);
```

Comment:
Why do you make this change?

Reply:
Change the log level so it's consistent with `JsonOutputter`.

https://github.com/apache/zookeeper/blob/master/zookeeper-server/src/main/java/org/apache/zookeeper/server/admin/JsonOutputter.java#L64